.chloggen/fix_jaeger-query-certs.yaml

```
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
+
+# The name of the component, or a single word describing the area of concern, (e.g. tempostack, tempomonolithic, github action)
+component: tempostack, tempomonolithic
```

```suggestion
component: tempostack
```

only affects tempostack, in tempomonolithic all runs in the same pod and therefore doesn't require mTLS